.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: rust
 
+branches:
+  only:
+  - master
+  - /*release*/
```

Comment:
Maybe it should be .* ?

Reply:
Actually yes. Travis treats these entries as (Ruby) regular expressions. So if you want to include branches like `0-9-1-release` then we should use something like `/-release$/` to match all branches ending with `-release`.

I think we should also add `/^v\d+\.\d+(\.\d+)?(-\S*)?$/` to the list as suggested by @dmitry-timofeev and Travis docs. This is to make sure that Travis builds our release tags which follow semver naming convention.

Reply:
I see, thanks. I added the ver regexp in our PR, just in case we ever have releases not in master (if that's not the case, it might be redundant as Travis is already set up to build `master`): https://github.com/exonum/exonum-java-binding/pull/484

Reply:
Fixed. :upside_down_face: